server/src/main/java/org/elasticsearch/rest/RestUtils.java

```diff
@@ -323,4 +328,21 @@ public static TimeValue getTimeout(RestRequest restRequest) {
         assert restRequest != null;
         return restRequest.paramAsTime(REST_TIMEOUT_PARAM, null);
     }
+
+    // Remove the BWC support for the deprecated ?local parameter.
+    // NOTE: ensure each usage of this method has been deprecated for long enough to remove it.
+    @UpdateForV10(owner = UpdateForV10.Owner.DISTRIBUTED_COORDINATION)
+    public static void consumeDeprecatedLocalParameter(RestRequest request) {
+        if (request.getRestApiVersion() == RestApiVersion.V_8) {
```

Comment:
we only test for equality because get component template didn't exist prior to version 8?

Reply:
`V_7` was removed from [`RestApiVersion`](https://github.com/elastic/elasticsearch/blob/78bd9ec6f082e67199bd1df742c72325b2169e2c/libs/core/src/main/java/org/elasticsearch/core/RestApiVersion.java#L19) :)